docs/src/description/constraints.md

```diff
@@ -165,30 +154,24 @@ integrity_constraints {
     # these are main constraints. they both express the same constraint.
     enf a' = a + 1;
     enf b' - b - 1 = 0;
-
-    # this is an auxiliary constraint, since it uses an auxiliary trace column.
-    enf p = p' * a;
 }
 ```
 
-### Periodic columns and random values
+### Periodic columns
 
-Integrity constraints can access the value of any periodic column in the current row, as well as random values provided by the verifier.
+Integrity constraints can access the value of any periodic column in the current row.
 
 To use periodic column values, the periodic column must be declared in the `periodic_columns` source section. The value in the current row can then be accessed by using the defined identifier of the periodic column.
 
-Random values can be accessed by using array indexing syntax on the `$rand` built-in, as described by the [accessor syntax rules](./syntax.md#section-specific-accessors).
-
-### Example of integrity constraints with periodic columns and random values
+### Example of integrity constraints with periodic columns
```

Comment:
Should we have a section with integrity constraints with busses? Or maybe just add busses to this section?